libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -605,7 +605,7 @@ void CompilerUtils::zeroInitialiseMemoryArray(ArrayType const& _type)
 		Whiskers templ(R"({
 			let size := mul(length, <element_size>)
 			// cheap way of zero-initializing a memory range
-			codecopy(memptr, codesize(), size)
```

Comment:
@axic will this make things worse for the ewasm translation? What are the rules for accessing code beyond its size?

Reply:
Any over read results in abort. So the current one or the proposed one both are failing.